onnxscript/rewriter/generic_pattern.py

```diff
@@ -294,6 +294,78 @@ def make_node(
         return proto
 
 
+class PatternMatchResult:
+    """Stores information about a match if a match was successful."""
+
+    def __init__(
+        self,
+        pattern: "GenericPattern",
+        model_nodes: typing.Sequence[oir.Node],
```

Comment:
```suggestion
        model_nodes: Sequence[oir.Node],
```

nit: I would import Sequence from typing instead, for simplicity

Reply:
I use typing everything and you always say not to do "from ... import ...". So I'm confused.

Reply:
For the full text: https://google.github.io/styleguide/pyguide.html#22-imports